haris2055/SOEN390
# Personalized search

This pull request includes the implementation of the following user stories: 

- #170 
- #122 
To test this feature:
  - go to the login page
  - sign in to a valid account
  - enter the search value in the search bar placed in the header
  - click the search button
  - **Expected Result**: The searched user and the searched jobs are displayed on the search screen. The user can select
 job section or user section to check the result. If there's no result matched, the message 'no result for $searchValue' will be displayed. The user should be able to access the user profile page and the job detail page by clicking button 'View profile' and 'View detail' separately. 

For filter:
 - On the search result page, click the button 'Add filter for job search'
 - A filter form should be displayed for the user to enter filter value
 - after filling the form, click apply
 - the searching result should be changed to match the filter value.

If merged, this pull request closes #122, closes #170.
